docs/pages/modules/use-create-expo-module-in-your-project.mdx

```diff
@@ -0,0 +1,179 @@
+import { StatusBar } from 'expo-status-bar';
+import React from 'react';
+import { Text, View } from 'react-native';
+
+export default function App() {
+  return (
+    <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
+      <Text>{Settings.hello()}</Text>
+      <StatusBar style="auto" />
+    </View>
+  );
+}
+```
+
+If you've configured everything correctly, you'll see a text saying "Hello world! 👋" in the app.
+
+## Publish the module to npm
+
+You can publish the module on npm and then add it to your project by installing it as a dependency.
+
+### 1. Initialize a new module
+
+Start by creating a new module using `create-expo-module`. Pay attention to each prompt as you'll publish this library, and choose a unique name for your npm package.
+
+<Terminal cmd={['$ npx create-expo-module expo-settings']} />
+
+### 2. Run the example project
+
+Run the example project to make sure everything is working. Then, start the TypeScript compiler to watch for changes and rebuild the JavaScript module.
+
+<Terminal
+  cmd={[
+    '# Run this in the root of the project to start the TypeScript compiler',
+    '$ npm run build',
+  ]}
+/>
+
+Open another terminal window, compile and run the example app:
+
+<Terminal
+  cmdCopy="cd example && npx expo run:ios"
+  cmd={[
+    '$ cd example',
+    '# Run the example app on Android',
+    '$ npx expo run:android',
+    '# Run the example app on iOS',
+    '$ npx expo run:ios',
+  ]}
+/>
+
+### 3. Publish your package to npm
+
+To publish your package to npm, you need to have an npm account. If you don't have one, create it on [the npm website](https://www.npmjs.com/signup). Once you have an account, log in using the following command:
+
+<Terminal cmd={['$ npm login']} />
+
+Then, navigate to the root of your module and publish your module running:
+
+<Terminal cmd={['$ npm publish']} />
+
+Your module will now be published to npm and can be installed in other projects using `npm install`.
+
+Apart from publishing your module to npm, there are other ways to use it in your project:
+
+- Creating a tarball of of your module using `npm pack` and install it directly in your project by running `npm install /path/to/tarball`. It's useful to test your module locally before publishing it to npm or if you want to share it with others who don't have access to the npm registry.
+- Running a local npm registry, such as [Verdaccio](https://verdaccio.org/), and installing your module from there. This is useful for cases where you want to host your own private npm registry to manage internal packages within a company or organization.
+- If you want to [publish a private package or use a private registry with EAS Builds](https://docs.expo.dev/build-reference/private-npm-packages/).
+
+If you want to publish a private private package or use a private registry with EAS Builds you can read how to configure it [here](https://docs.expo.dev/build-reference/private-npm-packages/).
```

Comment:
```suggestion
If you want to publish a private private package or use a private registry with EAS Builds you can read how to configure it [here](/build-reference/private-npm-packages/).
```

We always should relative links for internal pages, otherwise we perform full reload on navigation.

Reply:
updated